tests/plugins/autolink/manual/link.md

```diff
@@ -0,0 +1,10 @@
+@bender-ui: collapsed
+@bender-tags: 4.11.2, bug, 1824
```

Comment:
This test is brand new - could you explain why you decided to go with `4.11.2` tag instead of `4.15.2`?

Reply:
First: I read Comandeer comment about bumping and I thought that it should be bumped to the newest version. Then for some reason... I thought that if Jacek already bumped it - It should be ok like it was after his changes. 

I didn't think that it was 2 years ago and since it wasn't merged then, I should use the current version - so `4.15.2`.

Sorry for the confusion...

I change this to `4.15.2` again :)